backend/tests/unit/dal/Services/PropertyServiceTest.cs

```diff
@@ -54,11 +57,11 @@ public class PropertyServiceTest
                 new object[] { new AllPropertyFilter(50, 24, 50, 26), 0 },
                 new object[] { new AllPropertyFilter() { Agencies = new int[] { 3 } }, 7 },
                 new object[] { new AllPropertyFilter() { ClassificationId = 2 }, 2 },
-                new object[] { new AllPropertyFilter() { Description = "Description" }, 10 },
-                new object[] { new AllPropertyFilter() { Municipality = "Municipality" }, 1 },
+                new object[] { new AllPropertyFilter() { Description = "Description" }, 20 },
```

Comment:
what is the significance of the integers here, should they be unique?

Reply:
They represent the number of items the filter should return